example/40-validatingwebhookconfiguration.yaml

```diff
@@ -15,6 +15,7 @@ webhooks:
     - UPDATE
     resources:
     - shoots
```

Comment:
You could also update the name of the example ValidatingWebhookConfiguration from `gardener-extension-validator-azure` to `gardener-extension-admission-azure`.